README.md

```diff
@@ -365,3 +365,42 @@ is not initialized yet. Also you may want to stop using Vagrant and you want to
 wipe out all Vagrant control files and resources from your workstation. For those purposes you
 should consider using the `.ci/vagrant-cleaner.sh` script; run `.ci/vagrant-cleaner.sh -h` for
 further information.
+
+
+## Using the kata-local-ci libvirt-based script
+
+Another way to setup a local CI is tu use the `cmd/kata-local-ci` script. This
+script uses `libvirt` in order to setup a virtual machine in which the CI
+environment is created. This VM is then snapshotted immediately after setup, so
+that the same environment can be restored quickly.
+
+Compared to the `vagrant` approach, the main benefit is that this setup can
+easily be deployed on environment where `vagrant` is not available by default
+(like Red Hat Enteprise Linux or Red Hat Core OS systems). Also, because of the
+snapshotting, the setup is restored to the "clean" initial state at every run by
+default (this can be disabled by using the `-F` option).
+
+The VM snapshotting is particularly useful for people with a very slow network
+connexion, but will also save time for people with a reasonably fast one. In my
+measurements, the initial setup of the VM takes about 20 minutes on a reasonably
+fast network, but can take 3 hours (and fail with timeouts a large fraction of
+the runs) from a low-grade DSL.
+
+To get the various options provided by the script, use `cmd/kata-local-ci -h`.
+
+The most important options are:
+
+* `-d <distro>` to select a `fedora`, `ubuntu` or `centos` virtual machine. You
```

Comment:
Is `centos` still valid here or should it be stream/alma/rocky/scientific/blah?

Reply:
I added `alma` as a follow-up, see #4616 and https://github.com/c3d/kata-containers-tests/tree/issue/4261-local-ci. This is the branch where the update to this message will appear.